pkg/kubernetes/pods.go

```diff
@@ -53,3 +62,45 @@ func GetEnvVarIndex(envVarName string, env *[]corev1.EnvVar) int {
 	}
 	return 0
 }
+
+func GetOperatorPodOwnerRef(ns string, client crclient.Client) (*metav1.OwnerReference, error) {
+	// Get current Pod the operator is running in
+	pod, err := k8sutil.GetPod(context.TODO(), client, ns)
+	if err != nil {
+		return nil, err
+	}
+	podOwnerRefs := metav1.NewControllerRef(pod, pod.GroupVersionKind())
+	// Get Owner that the Pod belongs to
+	ownerRef := metav1.GetControllerOf(pod)
+	finalOwnerRef, err := findFinalOwnerRef(ns, ownerRef, client)
```

Comment:
When installed via OLM the CSV owns the Deployment. Does this produce any drawback? At first sight seems not...

Reply:
This code ported from the Operator SDK which is using for Service owner reference. Not sure that's will not work for Grafana Dashboard

Reply:
And we don't another alternatives way if Deployment name has been changed without labels which will be supported at least in the OCP 4.10
https://github.com/operator-framework/operator-lifecycle-manager/pull/2054

Reply:
It seems to me that Csv and Deployment have the same lifespan, so I think it's almost the same to have one or the other as owner for the dashboard object.